public/static/docs/api-reference/get_url.md

```diff
@@ -4,34 +4,43 @@ Returns the URL to the storage location of a data file or directory tracked in a
 <abbr>DVC project</abbr>.
 
 ```py
-dvc.api.get_url( path: str, repo: str = None,
-    rev: str = None, remote: str = None ) -> str
+def get_url(path: str,
+            repo: str = None,
+            rev: str = None,
+            remote: str = None) -> str
 ```
 
-**Usage:**
+#### Usage:
 
 ```py
 import dvc.api
 
-resource_url = dvc.api.get_url(
+dvc.api.get_url(
     'get-started/data.xml',
     repo='https://github.com/iterative/dataset-registry')
+
+# https://remote.dvc.org/dataset-registry/a3/04afb96060aad90176268345e10355
 ```
 
 ## Description
 
 Returns the URL string of the storage location (in a
 [DVC remote](/doc/command-reference/remote)) where a target file or directory,
-specified by its `path` in a `repo` (<abbr>DVC project</abbr>), is stored. If
-`repo` is not a DVC project, a `dvc.api.UrlNotDvcRepoError` is raised.
+specified by its `path` in a `repo` (<abbr>DVC project</abbr>), is stored.
 
 The URL is formed by reading the the project's
 [remote configuration](/doc/command-reference/config#remote) and the
-[DVC-file](/doc/user-guide/dvc-file-format) where the given `path` is an output
-(see the **Examples** below). The URL schema returned depends on the
+[DVC-file](/doc/user-guide/dvc-file-format) where the given `path` is an
+<abbr>output</abbr> The URL schema returned depends on the
 [type](/doc/command-reference/remote/add#supported-storage-types) of the
 `remote` used (see the [Parameters](#parameters) section).
 
+If the target is a directory, the returned URL will point to a file that
+contains the mapping of files in the directory (as a JSON array), along with
+their hash values. Refer to
+[Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
+and `dvc add` to learn more about how DVC handles data directories.
+
 ⚠️ This function does not check for the actual existence of the file or
 directory in the remote storage.
```

Comment:
I like this note but I think I had a similar one originally and we agreed it wasn't really about this function so we removed it. It's a little confusing given the note coming after (that actually who knows whether the file exists). Advanced DVC users should know this anyway if they need to know it? No strong opinion.

Reply:
Wait I see this was just moved from further down haha. Ok let me think... ⏳

Reply:
you didn't remove it actually )

Reply:
OK to move it but the original text is a little better I think, because it doesn't imply that the .dir file will exist (and contain "the mapping of files...").

Reply:
not sure I got the point ...

Reply:
I just wanted to avoid to many details (like .dir, etc - it's internal stuff)

Reply:
I think mentioning the .dir ending is appropriate as it may be something the user notices and wonders about when using this fn.

```suggestion
If the target is a directory, the returned URL will end in `.dir`: a file that
contains the mapping of files in the directory. Refer to
[Structure of cache directory](/doc/user-guide/dvc-files-and-directories#structure-of-cache-directory)
and `dvc add` to learn more about how DVC handles data directories.

⚠️ This function does not check for the actual existence of the file or
directory in the remote storage.
```

Reply:
they should not care about such specifics, we might change schema for those URL ... we can mention something like `(ends with `.dir`)` - but that should be enough

Reply:
> we can mention something like (ends with .dir) - but that should be enough 

Good idea. Keeping the last sentence too (link to dvc dir struct), I assume.